examples/llama-3/sppo-qlora-8b.yml

```
@@ -0,0 +1,74 @@
+base_model: meta-llama/Meta-Llama-3-8B-Instruct
+model_type: AutoModelForCausalLM
+tokenizer_type: AutoTokenizer
+
+load_in_8bit: false
+load_in_4bit: true
+strict: false
+
+rl: sppo
+rl_beta: 0.1
+datasets:
+  - path: orion-research/Aura-SPPO-Iter1_score
+    type: chatml.sppo_argilla_chat
+dataset_prepared_path:
+val_set_size: 0
+
+output_dir: ./outputs/out/Meta-Llama-3-8B-Instruct-SPPO-Iter1
+dataset_prepared_path: last_run_prepared
+
+adapter: qlora
+lora_model_dir:
+
+sequence_len: 1024
+sample_packing: false
+pad_to_sequence_len: true
+
+lora_r: 32
+lora_alpha: 16
+lora_dropout: 0.05
+lora_target_linear: true
+lora_fan_in_fan_out:
+lora_target_modules:
+peft_use_dora: true
+
+wandb_project:
+wandb_entity:
+wandb_watch:
+wandb_name:
+wandb_log_model:
+
+gradient_accumulation_steps: 8
+micro_batch_size: 1
+num_epochs: 1
+optimizer: paged_adamw_32bit
```

```suggestion
optimizer: paged_adamw_8bit
```
8bit optimizer is a bit friendlier for new users likely with smaller GPUs